bbb-libreoffice/install-bbb-libreoffice.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
Please review this entire `bbb-libreoffice` folder keeping only what's still needed ( I see network and systemd, both are not needed anymore ).

Reply:
All right, I removed all files that was used only in the previous approach!